src/windowsservercore/ltsc2022/helix/webassembly/amd64/Dockerfile

```diff
@@ -31,3 +31,8 @@ RUN npm install jsvu -g
 RUN npm exec -c "jsvu --os=win64 --engines=v8"
 RUN setx PATH "%PATH%;%USERPROFILE%\.jsvu\bin"
 RUN v8 -e "console.log(version());quit();"
+
+# install vc redistributable
+ENV VC_REDIST_VERSION 17
+RUN curl -SL --output %TEMP%\vc_redist.x64.exe https://aka.ms/vs/%VC_REDIST_VERSION%/release/vc_redist.x64.exe
+RUN %TEMP%\vc_redist.x64.exe /install /passive /norestart
```

Comment:
I think the `wait` option will be needed here to ensure the installer waits. Otherwise, it'll kick off and the container will exit while it's still running, leaving it an unfinished state.